.travis.yml

```diff
@@ -61,9 +58,6 @@ before_script:
       fi
   '
 
-env:
-  - TEST_BED_M2_HOME=/home/travis/.arquillian/mvn/apache-maven-3.3.9
```

Comment:
This was set because travis had older version of maven we couldn't use. It seems that since then it's using 3.5.0 on trusty image so that's a good find.

Reply:
Yeah, but the SWR is now using 3.5.2, that is not installed on the travis. Using the `embeddedMaven.useDefaultDistribution();` will guarantee that we are using the latest one SWR is compatible with

Reply:
So we cannot simply use anything lower than `3.5.x`? We might want to explore some different strategy to handle multiple mvn versions build in the future then.

Reply:
if your comment https://github.com/arquillian/smart-testing/pull/303#discussion_r162564945 is addressed, then sure, you can. 
see these lines - there is set a different Maven version https://github.com/arquillian/smart-testing/blob/master/functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/ProjectBuilder.java#L127-L129